src/ExadelMentorship.BusinessLogic/Features/WeatherFeature/CurrentWeatherCommand.cs

```diff
@@ -13,30 +13,16 @@
 
 namespace ExadelMentorship.BusinessLogic.Features.WeatherFeature
 {
-    public class Weather : IWeather
+    public class CurrentWeatherCommand : ICurrentWeatherCommand, ICommand
```

Comment:
This is a DTO class and shouldn't provide any service. Remove `ICurrentWeatherCommand` interface and move its code (i.e. as a private methdo) to the handler, as the handler basically replaces this it.

Reply:
class 'CurrentWeatherCommand', has only one method 'GetTemperatureByCityName'. If I will move this method to Handler class, then nothing will be inside 'CurrentWeatherCommand' class. Why we need it at all?

Reply:
Nice question. You already know that in C# an interface define a contract. Since we are trying to implement command objects, now a command object defines a contract! When an objects acts as a contract, this gives a flexibility to have more contracts without introducing new interfaces.